Dockerfile

```diff
@@ -90,20 +93,42 @@ RUN pip3 install -r requirements.txt -c constraints.txt
 RUN gem install bundler && bundle install
 RUN apt-get remove -y build-essential ruby-dev
 
+# nginx unprivileged setup
+RUN ln -sf /dev/stdout /var/log/nginx/access.log && \
+    ln -sf /dev/stderr /var/log/nginx/error.log && \
+    sed -i '/user www-data;/d' /etc/nginx/nginx.conf && \
+    sed -i 's,/run/nginx.pid,/tmp/nginx.pid,' /etc/nginx/nginx.conf && \
+    sed -i "/^http {/a \    proxy_temp_path /tmp/proxy_temp;\n    client_body_temp_path /tmp/client_temp;\n    fastcgi_temp_path /tmp/fastcgi_temp;\n    uwsgi_temp_path /tmp/uwsgi_temp;\n    scgi_temp_path /tmp/scgi_temp;\n" /etc/nginx/nginx.conf && \
```

Comment:
Perhaps cleaner to include the modified `nginx.conf` file in the image instead of manipulating it on the fly? Otherwise, a comment explaining what each command does will be helpful.

Reply:
This is based on `nginxinc/docker-nginx-unprivileged`: https://github.com/nginxinc/docker-nginx-unprivileged/blob/main/Dockerfile-debian.template

Reply:
IMO overriding this way seems better than having a config rot. Because this updates the default configuration comes with nginx OS distribution.